sdk/cosmos/azure-cosmos/azure/cosmos/_execution_context/aio/endpoint_component.py

```diff
@@ -173,7 +173,7 @@ def __init__(self, execution_context, aggregate_operators):
         for operator in aggregate_operators:
             if operator == "Average":
                 self._local_aggregators.append(_AverageAggregator())
-            elif operator == "Count":
+            elif operator in ("Count", "CountIf"):
```

Comment:
There are no changes to _CountAggregator in this PR - how is this supposed to start supporting CountIf?